pawn/Resources/Callbacks/OnPlayer/OnPlayerCommandText.pwn

```diff
@@ -873,11 +877,19 @@ public OnPlayerCommandText(playerid, cmdtext[]) {
                 player, playerLevel);
 
             // If the user was temp'd, show admins who temp'd the player.
-            if (tempLevel[player] == 1 || tempLevel[player] == 2) {
```

Comment:
What is your reason to swap these ifs arounds?

Reply:
Isn't it right?